progression/src/main/resources/progression-migrations/V1_1__progression.sql

```diff
@@ -24,6 +24,14 @@ create table if not exists progression_mining
     primary key (Gamer, Material)
 );
 
+
+create table if not exists progression_woodcutting
+(
+    Gamer varchar(36) not null,
+    Material varchar(36) not null,
+    primary key (Gamer, Material)
+);
```

Comment:
In this state, the table isn't really required since it doesnt store anything useful

Same problem we have with the mining table at the moment, I think we could do a single row per log chopped

Gamer, Material, Location (String), Time
Queries will be quite efficient